content/en/docs/guide_openapi/managing_apidocs.md

```diff
@@ -0,0 +1,68 @@
+
+The details in the following sections are only relevant if operations are bound to flows. If operations are not bound to flows, you can update them without any affect.
+
+### Changing a bound operationId
+
+Changing the name of a bound operation's `operationId` is the same as deleting it and adding a new one with a new name. It is impossible to distinguish. When this happens, the old operation and flow will become unbound. To fix, edit the flow, and change the **OpenAPI** flow-trigger **Method** to use the new `operationId`.
+
+### Removing a bound operation
+
+This is an unsafe update. If you remove an [OpenAPI operation](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#operationObject) that is bound to a flow, the flow will become "unbound", and the console window and UI will show an error that this happened. If you have no need of the flow, it is [safe to delete](/docs/developer_guide/flows/manage_flows#delete-flows).
+
+### Changing method, path, or path parameter of a bound operation
+
+If your operation does not use an [`operationId`](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#operation-object), then changing the method, path, or path parameters of a bound operation is the same as [removing a bound operation](#removing-a-bound-operation). This is discussed in the [writing OpenAPI documents](/docs/guide_openapi/writing_apidocs#use-operationid) on how to manage your API more effectively. However, if your operation does use an [`operationId`](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#operation-object), then it is shielded against this.
+
+### Changes that may break a flow
+
+In addition to directly manipulating the method, path, or path parameters of an [Operation object](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#operationObject), the following changes may break your flow and require additional changes in the flow editor.
+
+* Removing parameters
+* Renaming parameters (this is effectively removing them)
+* Changing parameter names (even capitalization)
+* Changing the request
+* Changing the response content-type or schema
+* Removing responses
+
+## Deleting OpenAPI documents
+
+Deleting you OpenAPI documents is one of these actions that should be taken with caution. It is an irreversible action that will have a big impact on your service as you are effectively removing a chunk from it. It is important to note that deleting an OpenAPI document will delete all associated flows too. Any non related flows will be preserved.
+
+TODO: Insert gif here showcasing:
```

Comment:
TODO: When we come up with the exact process of creating these gifs, add a gif here showcasing the details underneath.

I've attempted to do this with screenshots, but it adds a lot of additional text and space with the screenshots itself, so a simple gif will be best.